cleanlab/count.py

```diff
@@ -1449,7 +1449,8 @@ def get_confident_thresholds(
             np.mean(pred_probs[:, k][labels == k]) if k in unique_classes else BIG_VALUE
             for k in all_classes
         ]
-        return np.array(confident_thresholds)
+        confident_thresholds = np.clip(confident_thresholds, 2e-6, 1.0)
```

Comment:
We do not want a bunch of constants like this floating around throughout the codebase. I think we should address this via proper engineering. 

The proper way to do things is to define a `constants.py` file somewhere in `internal/`

And then define this as a variable in that file like this: ```CONFIDENT_THRESHOLDS_LOWER_BOUND = 2e-6  # lower bound imposed to clip confident thresholds from below, not 1e-6 because of the floating point comparison
```
and then we should import this variable in all files where it's needed:

```
from cleanlab.internal.constants import CONFIDENT_THRESHOLDS_LOWER_BOUND 
```